docs/reference/primitives/rack/registry.md

```diff
@@ -0,0 +1,19 @@
+# Registry
+
+A reference to an external private Docker registry that you can reference in your [Builds](../app/build.md).
```

Comment:
double "reference" is a bit awkward and external/private is a bit redundant.

Maybe "A Registry is a set of stored credentials for a private Docker registry that can be referenced during a Build"